userguide/advanceduse/ssh.rst

```diff
@@ -50,6 +50,13 @@ Connect
 Now everything is set up and you can use ``ssh`` ::
 
     ssh phablet@<ip-address>
+    
+To identify the ip-address of your UT devce,open Termux application in your phone and run the following command::
```

Comment:
(I could have sworn I already posted this ... but it is nowhere to be seen ... oh well, here it goes again)

devce -> device
space after the comma
I think Termux is some android app. I guess you mean Terminal

Reply:
what about this?